### ISSUE TITLE
<08_09_2023><NYU><ANDROID><Accessibility_Update Profile Screen><Proper Alignment should be there when user select "Name Column" and "Gender" ><Parent_ID-1980>

### ISSUE THREAD

**Issue by 123ajay1994** [nammayatri/nammayatri#3063]
#1980 

**Describe the bug**
Proper Alignment should be there when user select "Name Column" and "Gender", While selecting "Gender" it is little cropping and while selecting 

**To Reproduce**
Steps to reproduce the behavior.

**Expected behavior**

**Screenshots**
https://drive.google.com/file/d/1h6X7zcbe0dFl7g8SAlTgXoj-1uyr8kMF/view?usp=sharing
https://drive.google.com/file/d/1LjnsWsabU6nI8hFLMRLUTWYqa3MmXQOC/view?usp=sharing
**App (please complete the following information):**

**Additional context**

**Comment by MercyQueen** [comment]
It's not cropping. Its the padding which is added on top of the primary button so that the user feels easy to differentiate between the button and other components on screen
